src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -97,17 +98,19 @@ public void target() {
 
   /** Starts the roller. */
   public void start() {
+    System.out.println("Starting roller");
     mRoller.setSpeed(ROLLER_SPEED);
   }
 
   /** Stops the roller. */
   public void stop() {
+    System.out.println("Stopping roller");
     mRoller.setSpeed(0);
   }
 
   /** Returns whether roller is at full speed. */
   public boolean isAtVelocity() {
-    return mRollerVelocity > TARGET_ROLLER_VELOCITY;
+    return true;// mRollerVelocity > TARGET_ROLLER_VELOCITY;
```

Comment:
Why do we want this to always return true now?

Reply:
https://github.com/Team138Entropy/Entropy2020/issues/99